integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/DpvConversationStrategy.java

```diff
@@ -49,7 +49,7 @@ public ResponseEntity send(ConversationResource conversationResource) {
         ServiceRecord serviceRecord = sr.getServiceRecord(cr.getReceiverId());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "serviceRecord" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "serviceRecord". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1854)